Lib/uuid.py

```diff
@@ -700,24 +721,35 @@ def uuid3(namespace, name):
     """Generate a UUID from the MD5 hash of a namespace UUID and a name."""
     if isinstance(name, str):
         name = bytes(name, "utf-8")
-    from hashlib import md5
-    digest = md5(
-        namespace.bytes + name,
-        usedforsecurity=False
-    ).digest()
-    return UUID(bytes=digest[:16], version=3)
+    # HACL*-based MD5 is slightly faster than its OpenSSL version,
```

Comment:
I would put these comments in the pr comments and leave them out of the code. (but I do see the value of them)

Reply:
Done